packages/bpk-component-content-cards/src/ContentCard/ContentCard.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import BpkText from 'bpk-component-text';
+import { cssModules } from 'bpk-react-utils';
+
+import STYLES from './ContentCard.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+type CardLayout = 'HORIZONTAL' | 'VERTICAL';
+
+type Props = {
+  card: {
+    imageLink?: string;
+    headline?: string;
+    description?: string;
+    ctaLink?: string;
+  };
+  layout: CardLayout;
+};
+
+const ContentCard = ({ card, layout }: Props) => (
+  <a
+    className={[
+      getClassName('bpk-content-cards__card-link'),
+      layout === 'HORIZONTAL'
+        ? getClassName('bpk-content-cards__card-horizontal')
+        : getClassName('bpk-content-cards__card-vertical'),
+    ].join(' ')}
```

Comment:
You can simplify this by doing

```suggestion
    className={
      getClassName('bpk-content-cards__card-link',
      layout === 'HORIZONTAL'
        ? 'bpk-content-cards__card-horizontal'
        : 'bpk-content-cards__card-vertical',
    )}
```